src/main/Port.Adapter/UI/Maui/ViewModels/CreateAvatarViewModel.cs

```diff
@@ -20,6 +21,7 @@ public partial class CreateAvatarViewModel : BaseViewModel
     private readonly LogController logController = new();
     private readonly IAvatarApplicationService avatarApplicationService;
     private readonly IProgressService progressService;
+    private AvatarServerConfiguration AvatarServerConfiguration = new(string.Empty);
```

Comment:
Field AvatarServerConfiguration should use camel casing with a lowercase first letter.